src/manifolds/Rotations.jl

```diff
@@ -240,37 +240,40 @@ end
 
 flat!(M::Rotations, ξ::CoTFVector, p, X::TFVector) = copyto!(ξ, X)
 
-function get_coordinates(M::Rotations, p, X, B::ArbitraryOrthonormalBasis) where {N}
-    T = Base.promote_eltype(p, X)
-    return vee(M, p, X) .* sqrt(T(2))
-end
-
-function get_vector(M::Rotations, p, X, B::ArbitraryOrthonormalBasis) where {N}
-    T = Base.promote_eltype(p, X)
-    return hat(M, p, X) ./ sqrt(T(2))
-end
-
 @doc raw"""
-    hat(M::Rotations, p, Xⁱ)
+    get_vector(M::Rotations, p, Xⁱ)
```

Comment:
I'd prefer for `Rotations` that we use the more specific default basis `DefaultOrthogonalBasis`. That the basis is orthogonal is important for its use, and of course the behavior will be the same.
```suggestion
    get_vector(M::Rotations, p, Xⁱ, B:: DefaultOrthogonalBasis)
```

Reply:
Here and below

Reply:
I will do the ones below later, since I am currently at the airport and have to first finish something on another branch then.